cryptography-core/src/asset_proofs/membership_proof.rs

```diff
@@ -4,6 +4,7 @@
 //! This implementation is based on one-out-of-many proof construction described in the following paper
 //! <https://eprint.iacr.org/2015/643.pdf>
 
+use super::errors::{ErrorKind, Fallible};
```

Comment:
Why `super` vs `crate`? This is a general question not specifically for this line.

Reply:
`crate` refers to the root scope vs `super` refers to the parent scope. here we could do `crate::asset_proofs::errors::{ErrorKind, Fallible};` instead.